proto/minder/v1/minder.proto

```diff
@@ -2559,70 +2559,73 @@ message ListEvaluationHistoryResponse {
 }
 
 message EvaluationHistory {
-    // entity contains details of the entity which was evaluated.
-    EvaluationHistoryEntity entity = 1;
+    message Entity {
+        // id is the ID of the entity.
+        string id = 1;
 
-    // rule contains details of the rule which the entity was evaluated against.
-    EvaluationHistoryRule rule = 2;
+        // type is the entity type.
+        minder.v1.Entity type = 2;
```

Comment:
By naming the message `Entity`, you're conflicting with the existing top-level definition of `Entity` elsewhere in the file, which seems like it may _also_ make the generated code more difficult to work with.

Reply:
The naming conflict is only visible at this very line of the protobuf definitions, and generated go types are totally unaffected. The last commit I pushed to this PR should clarify the impact on go code.

The reasoning behind embedding messages was that they're only meant to be used in this context and nowhere else. I don't think we want to use them anywhere else in the API, so I don't understand the point of having them as top-level messages.

I will revert the last commit and avoid embedding as it was a breaking change anyway.